### PR TITLE
Serve room composite templates locally

### PR DESCRIPTION
**Pull request by biglittlebigben**

### CODE REVIEW

**Comment by biglittlebigben** [comment]
Build will fail until we push the new template image to dockerhub

**Review by frostbyte73 — APPROVED** [review]
lgtm, let's publish before merging

**Comment by frostbyte73** [comment]
should probably change .github/workflows/deploy-web to publish instead of deploy

File: .github/workflows/deploy-web.yaml [hunk 1]
```diff
@@ -1,47 +1,36 @@
...
-          yarn exec gh-pages -- -d build -u "github-actions-bot <support+actions@github.com>" --no-history
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./build/template/Dockerfile
+          push: true
+          platforms: linux/amd64
```
Comment (frostbyte73) [inline .github/workflows/deploy-web.yaml:34]: arm64 as well?

**Pull request merged**